prototypes/recipes/mukmoux/recipes-mukmoux.lua

```diff
@@ -264,11 +401,90 @@ RECIPE {
+        {type = 'item', name = 'mukmoux-mk03', amount = 1, probability = 0.7},
+    },
+    --main_product = "mukmoux-mk04",
+    icons =
+        {
+            {icon = '__pyalienlifegraphics__/graphics/icons/evolution-mk04.png', icon_size = 64},
+            {icon = "__pyalienlifegraphics__/graphics/icons/mukmoux.png", icon_size = 64, scale = 0.25,shift = {-7.5,-7.5}},
+        },
+    icon_size = 64,
+    subgroup = 'py-alienlife-mukmoux',
+    order = 'zca'
+}:add_unlock("mukmoux-breeding-3")
+
+RECIPE {
+    type = 'recipe',
+    name = 'mukmoux-mk04-breeder',
+    category = 'mukmoux',
+    enabled = false,
+    energy_required = 30,
+    ingredients = {
+        {type = 'item', name = 'mukmoux-mk04', amount = 2},
+        {type = 'item', name = 'mukmoux-food-02', amount = 2},
+        {type = 'item', name = 'bedding', amount = 4},
+        {type = 'item', name = 'moss', amount = 10},
+        {type = 'item', name = 'saps', amount = 10},
+        {type = 'item', name = 'water-barrel', amount = 3},
+        },
+    results = {
+        {type = 'item', name = 'mukmoux-calf-mk04', amount = 5},
+        {type = 'item', name = 'empty-barrel', amount = 3},
+    },
+    --main_product = "mukmoux-calf-mk04",
+    icons =
+    {
+        {icon = "__pyalienlifegraphics__/graphics/icons/mukmoux.png"},
+        {icon = '__pyalienlifegraphics__/graphics/icons/evolution-mk04.png', scale = 0.25,shift = {-7.5,-7.5}},
+    },
+    icon_size = 64,
+    subgroup = 'py-alienlife-mukmoux',
+    order = 'zcb'
+}:add_unlock("mukmoux-breeding-3")
+
+RECIPE {
+    type = 'recipe',
+    name = 'mukmoux-calf-mk04-breeder',
+    category = 'mukmoux',
+    enabled = false,
+    energy_required = 30,
+    ingredients = {
+        {type = 'item', name = 'mukmoux-calf-mk04', amount = 4},
+        {type = 'item', name = 'mukmoux-food-02', amount = 2},
+        {type = 'item', name = 'bedding', amount = 4},
+        {type = 'item', name = "seaweed", amount = 10},
+        {type = 'item', name = 'saps', amount = 10},
+        {type = 'item', name = 'barrel-milk', amount = 2},
+        {type = 'item', name = 'water-barrel', amount = 3},
+        },
+    results = {
+        {type = 'item', name = 'mukmoux-mk04', amount_min = 2, amount_max = 4},
+        {type = 'item', name = 'empty-barrel', amount = 3},
+        {type = 'item', name = 'empty-barrel-milk', amount = 2},
     },
     --main_product = "mukmoux-mk04",
-    icon = '__pyalienlifegraphics__/graphics/icons/evolution-mk04.png',
+    icons =
+        {
+            {icon = "__pyalienlifegraphics__/graphics/icons/mukmoux-calf.png"},
+            {icon = '__pyalienlifegraphics__/graphics/icons/evolution-mk04.png', scale = 0.25,shift = {-7.5,-7.5}},
+        },
     icon_size = 64,
     subgroup = 'py-alienlife-mukmoux',
-    order = 'zc'
-}:add_unlock("biased-mutation")
+    order = 'zcc'
+}:add_unlock("mukmoux-breeding-3")
+
+
+table.insert(data.raw.module["mukmoux"].limitation, "mukmoux-mk02")
```

Comment:
Would help to add code that handles all module limitations for PyAL in a consistent way. Probably similar to data-updates.lua:83 where modules are enabled based on the recipe category.